ulib/experimental/Steel.Reference.fst

```diff
@@ -209,101 +209,117 @@ let cas_pt #t #uses r v v_old v_new =
                  (fun _ -> ());
   return b
 
-(* Simple Reference library, only full permissions.
-   AF: Permissions would likely need to be an index of the vprop ptr.
+(* Simple Reference library.
+   AF: Permissions need to be an index of the vprop ptr.
```

Comment:
Initials can be removed here, and "simple reference library" should be renamed into something like "Library for references with fractional permissions"